### ISSUE TITLE
Modals not showing in production build

### ISSUE THREAD

**Issue by johanquiroga** [adeptoas/sweet-modal-vue#67]
Hello,

I have installed the module globally and everything runs smoothly in development environment, like this:

![tasks-app-dev](https://user-images.githubusercontent.com/22845267/37879314-8e589bcc-303c-11e8-997e-72c2e856e2e9.gif)

But in production version the modals aren't showing at all:

![tasks-app-prod](https://user-images.githubusercontent.com/22845267/37879333-dd5f1cfa-303c-11e8-9486-9e7172381c19.gif)

And the following error appears in console:
![image](https://user-images.githubusercontent.com/22845267/37879342-123cadd4-303d-11e8-90ca-e9768b5f6997.png)

I would like some help with this issue, if someone knows why its happening.

The repo can be found [here](https://github.com/johanquiroga/tasks-app) and production version is deployed [here](http://tasks-app.johanquiroga.me)

Thanks!

**Comment by siquick** [comment]
Having the same issue - did you get this working @johanquiroga ?

**Comment by johanquiroga** [comment]
@siquick No, I couldn't find what was the problem so I just used [this](https://www.npmjs.com/package/vue-sweetalert2) package instead
